questionnaire/routes.js

```diff
@@ -236,10 +241,37 @@ router
             });
             const sectionId = formHelper.addPrefix(req.params.section);
             const body = formHelper.processRequest(req.body, req.params.section);
+            // eslint-disable-next-line no-underscore-dangle
+            const pageExternalId = body._externalId;
+            if (
+                isAuthenticated &&
+                pageExternalId !== undefined &&
+                req.session?.externalId !== undefined &&
+                pageExternalId !== req.session?.externalId
+            ) {
+                if (!validateExternalId(pageExternalId)) {
+                    console.info(
+                        `Malformed page external id received for questionnaire ${getQuestionnaireIdInSession(
+                            req.session
+                        )}. Redirecting to dashboard.`
+                    );
+                } else {
+                    console.info(
+                        `External id ${
+                            req.session?.externalId
+                        } for questionnaire id ${getQuestionnaireIdInSession(
```

Comment:
put the actual GUIDs in quotes within the string to make it more readable.

i.e.:

````
'External id "urn:uuid:ce66be9d-5880-4559-9a93-df15928be396" for questionnaire id "c992d660-d1a8-4928-89a0-87d4f9640250" does not match external id "urn:uuid:11111111-2222-4333-8444-555555555555" on page. Redirecting to dashboard.'
````